packages/plugins/rubygem-foreman-tasks/rubygem-foreman-tasks.spec

```diff
@@ -135,8 +135,8 @@ type foreman-selinux-relabel >/dev/null 2>&1 && foreman-selinux-relabel 2>&1 >/d
 %{foreman_assets_foreman}
 %{gem_instdir}/public/assets/foreman_tasks/
 %{foreman_dir}/public/assets/foreman_tasks
-%{foreman_webpack_plugin}
-%{foreman_webpack_foreman}
```

Comment:
If this works, we should update the macro in `foreman.spec` to only use underscores. Because then the lines above are probably also redundant.

Reply:
no, the lines above are sprockets, and those actually do generate both, `foreman_tasks` *and* `foreman-tasks` assets (lovely, huh?)
```
** Execute plugin:assets:precompile
I, [2024-01-29T18:11:35.341591 #79]  INFO -- : Writing /builddir/build/BUILDROOT/rubygem-foreman-tasks-9.0.2-2.fm3_10.el8.x86_64/usr/share/gems/gems/foreman-tasks-9.0.2/public/assets/foreman_tasks/trigger_form-819977735d4974feaf40cb92589b85cbe3f5b19e5794f8d31debb1501031a83d.css
I, [2024-01-29T18:11:35.342143 #79]  INFO -- : Writing /builddir/build/BUILDROOT/rubygem-foreman-tasks-9.0.2-2.fm3_10.el8.x86_64/usr/share/gems/gems/foreman-tasks-9.0.2/public/assets/foreman_tasks/trigger_form-819977735d4974feaf40cb92589b85cbe3f5b19e5794f8d31debb1501031a83d.css.gz
I, [2024-01-29T18:11:35.342417 #79]  INFO -- : Writing /builddir/build/BUILDROOT/rubygem-foreman-tasks-9.0.2-2.fm3_10.el8.x86_64/usr/share/gems/gems/foreman-tasks-9.0.2/public/assets/foreman_tasks/foreman_tasks-819977735d4974feaf40cb92589b85cbe3f5b19e5794f8d31debb1501031a83d.css
I, [2024-01-29T18:11:35.342609 #79]  INFO -- : Writing /builddir/build/BUILDROOT/rubygem-foreman-tasks-9.0.2-2.fm3_10.el8.x86_64/usr/share/gems/gems/foreman-tasks-9.0.2/public/assets/foreman_tasks/foreman_tasks-819977735d4974feaf40cb92589b85cbe3f5b19e5794f8d31debb1501031a83d.css.gz
I, [2024-01-29T18:11:35.343145 #79]  INFO -- : Writing /builddir/build/BUILDROOT/rubygem-foreman-tasks-9.0.2-2.fm3_10.el8.x86_64/usr/share/gems/gems/foreman-tasks-9.0.2/public/assets/foreman_tasks/trigger_form-20a7310e97386aba5900b38cf2da93b81dddc2047cf215a8fc031317adf37720.js
I, [2024-01-29T18:11:35.343655 #79]  INFO -- : Writing /builddir/build/BUILDROOT/rubygem-foreman-tasks-9.0.2-2.fm3_10.el8.x86_64/usr/share/gems/gems/foreman-tasks-9.0.2/public/assets/foreman_tasks/trigger_form-20a7310e97386aba5900b38cf2da93b81dddc2047cf215a8fc031317adf37720.js.gz
I, [2024-01-29T18:11:35.344095 #79]  INFO -- : Writing /builddir/build/BUILDROOT/rubygem-foreman-tasks-9.0.2-2.fm3_10.el8.x86_64/usr/share/gems/gems/foreman-tasks-9.0.2/public/assets/foreman_tasks/foreman_tasks-20a7310e97386aba5900b38cf2da93b81dddc2047cf215a8fc031317adf37720.js
I, [2024-01-29T18:11:35.344514 #79]  INFO -- : Writing /builddir/build/BUILDROOT/rubygem-foreman-tasks-9.0.2-2.fm3_10.el8.x86_64/usr/share/gems/gems/foreman-tasks-9.0.2/public/assets/foreman_tasks/foreman_tasks-20a7310e97386aba5900b38cf2da93b81dddc2047cf215a8fc031317adf37720.js.gz
I, [2024-01-29T18:11:35.345057 #79]  INFO -- : Writing /builddir/build/BUILDROOT/rubygem-foreman-tasks-9.0.2-2.fm3_10.el8.x86_64/usr/share/gems/gems/foreman-tasks-9.0.2/public/assets/foreman-tasks/locale/zh_TW/foreman_tasks-2903b39e63207354c8b3cd6c33206d35c1221ab11a62e684db84af7604fe1ff3.js
I, [2024-01-29T18:11:35.345243 #79]  INFO -- : Writing /builddir/build/BUILDROOT/rubygem-foreman-tasks-9.0.2-2.fm3_10.el8.x86_64/usr/share/gems/gems/foreman-tasks-9.0.2/public/assets/foreman-tasks/locale/zh_TW/foreman_tasks-2903b39e63207354c8b3cd6c33206d35c1221ab11a62e684db84af7604fe1ff3.js.gz
I, [2024-01-29T18:11:35.345733 #79]  INFO -- : Writing /builddir/build/BUILDROOT/rubygem-foreman-tasks-9.0.2-2.fm3_10.el8.x86_64/usr/share/gems/gems/foreman-tasks-9.0.2/public/assets/foreman-tasks/locale/zh_CN/foreman_tasks-534335c104bf41f9c20b51f1dacc0f32bc0beea69651096f836b5477dc9faf68.js
I, [2024-01-29T18:11:35.346864 #79]  INFO -- : Writing /builddir/build/BUILDROOT/rubygem-foreman-tasks-9.0.2-2.fm3_10.el8.x86_64/usr/share/gems/gems/foreman-tasks-9.0.2/public/assets/foreman-tasks/locale/zh_CN/foreman_tasks-534335c104bf41f9c20b51f1dacc0f32bc0beea69651096f836b5477dc9faf68.js.gz
I, [2024-01-29T18:11:35.347082 #79]  INFO -- : Writing /builddir/build/BUILDROOT/rubygem-foreman-tasks-9.0.2-2.fm3_10.el8.x86_64/usr/share/gems/gems/foreman-tasks-9.0.2/public/assets/foreman-tasks/locale/ru/foreman_tasks-aa9739f89f7811b82330f60822fa7d6f7cc9bcb1c4a8e32141dff225958e9a5f.js
```

Reply:
As always, using a dash for a plugin name was a mistake.

Reply:
But doesn't it still remain? `foreman_webpack_plugin` should now always use underscores for plugin identifiers, not `%{gem_name}`:
https://github.com/theforeman/foreman-packaging/blob/35cdc9224547c1556131c8b1b0cc787c8d44b6b7/packages/foreman/foreman/foreman.spec#L688-L690

Reply:
I don't think so. There's `plugin_name` which comes close. We could change that to `foreman_tasks`. It would only change the `Provides: foreman-plugin-foreman-tasks` to `Provides: foreman-plugin-foreman_tasks` but I think that's ok. We can even manually keep the old entry.

Reply:
`plugin_name` is also used for asset compilation etc, so I'd rather not change that right now